DOC-2826 add a note (#3004)
RedisLabs/redislabs-docs: content/embeds/port-53.md

    sudo ln -s /run/systemd/resolve/resolv.conf /etc/resolv.conf
    ```

    {{< note >}}

> You may encounter a temporary name resolution error (`sudo: unable to resolve host {hostname}: Temporary failure in name resolution`) which should be fixed by restarting `systemd-resolved` in the next step.

Reviewer: Maybe some slight changes
```suggestion
You might encounter a temporary name resolution error (`sudo: unable to resolve host {hostname}: Temporary failure in name resolution`), which should be fixed when you restart `systemd-resolved` in the next step.
```
